Sources/PointFreeRouter/Routes.swift

```diff
@@ -63,6 +63,7 @@ public enum Route: DerivePartialIsos, Equatable {
 
   public enum Invite: DerivePartialIsos, Equatable {
     case accept(TeamInvite.Id)
+    case addTeammate(EmailAddress?)
```

Comment:
I added the route here, but maybe it makes sense to be in `Route.Account.Subscription.Change` instead? what do you think?

Reply:
i think here is good. `Change` should probably go away soon in favor of explicit `upgrade` and `downgrade` routes. would simplify a lot